README.md

```diff
@@ -2,20 +2,18 @@
 
 JupyterLab Desktop is the cross-platform desktop application for [JupyterLab](https://github.com/jupyterlab/jupyterlab). It is the quickest and easiest way to get started with Jupyter notebooks on your personal computer, with the flexibility for advanced use cases.
 
-![JupyterLab Desktop](media/jupyterlab-desktop.png)
 
-## Installation
+![JupyterLab Desktop](media/jupyterlab-desktop.png)
 
-If you have an existing JupyterLab Desktop installation, please uninstall it first by following the [uninstall instructions](user-guide.md#uninstalling-jupyterlab-desktop).
+## Who is it for?
 
-- [Debian, Ubuntu Linux Installer](https://github.com/jupyterlab/jupyterlab-desktop/releases/latest/download/JupyterLab-Setup-Debian.deb)
```

Comment:
This is separate work and probably requires a separate landing page... but could we setup a webpage that reads the browser user-agent and suggests/auto-starts the most relevant download.  I think the benefit here is less distraction more than it is a user not knowing what OS they are on.